tests/UnifiedSpecTests/Operation.php

```diff
@@ -579,7 +592,28 @@ private function executeForDatabase(Database $database)
                 return iterator_to_array($database->listCollectionNames($args));
 
             case 'listCollections':
-                return iterator_to_array($database->listCollections($args));
+                return array_map(
+                    function (CollectionInfo $info) {
+                        return $info->__debugInfo();
+                    },
+                    iterator_to_array($database->listCollections($args))
+                );
+
+            case 'modifyCollection':
+                assertArrayHasKey('collection', $args);
+                assertIsString($args['collection']);
+
+                /* ModifyCollection takes collection and command options
+                 * separately, so we must split the array after initially
+                 * filtering out the collection name.
+                 *
+                 * The typeMap option is intentionally omitted since it is
+                 * specific to PHPLIB and will never appear in spec tests. */
+                $options = array_diff_key($args, ['collection' => 1]);
+                $collectionOptions = array_diff_key($options, ['session' => 1, 'writeConcern' => 1]);
+                $options = array_intersect_key($options, ['session' => 1, 'writeConcern' => 1]);
+
+                return $database->modifyCollection($args['collection'], $collectionOptions, $options);
```

Comment:
@jmikola It would be helpful to add the description of new option "changeStreamPreAndPostImages" inside "Supported options" section of the operating class (ModifyCollection.php):
https://github.com/jmikola/mongo-php-library/blob/phplib-843/src/Operation/ModifyCollection.php
And also it looks like this new option needs to be described in the doc file for modifyCollection method:
https://github.com/jmikola/mongo-php-library/blob/phplib-843/docs/includes/apiargs-MongoDBDatabase-method-modifyCollection-option.yaml

Reply:
I see you also brought this up in https://github.com/mongodb/mongo-php-library/pull/928#discussion_r874230058. See that thread for my response, since this question pertains to PHPLIB-856.

Note: this PR is only for PHPLIB-874 and its branch is rebased on #928 due to the dependency. Only the last commit is relevant, and this PR will be reduced to a single commit once #928 is merged.